parl/env/atari_wrappers.py

```diff
@@ -27,6 +27,8 @@
 
 cv2.ocl.setUseOpenCL(False)
 
+__all__ = ['wrap_deepmind']
```

Comment:
'wrap_deepmind', 'MonitorEnv', 'get_wrapper_by_cls' , refer to https://github.com/PaddlePaddle/PARL/blob/develop/examples/A2C/actor.py#L21

Reply:
done